platforms/win32/vm/sqWin32Main.c

```diff
@@ -925,41 +879,20 @@ sqInt  fileHandleType(HANDLE fdHandle) {
 	int size = sizeof(FILE_NAME_INFO) + sizeof(WCHAR) * MAX_PATH;
 	FILE_NAME_INFO *nameinfo;
 	WCHAR *p = NULL;
-
-	typedef BOOL(WINAPI *pfnGetFileInformationByHandleEx)(
-		HANDLE                    hFile,
-		FILE_INFO_BY_HANDLE_CLASS FileInformationClass,
-		LPVOID                    lpFileInformation,
-		DWORD                     dwBufferSize
-		);
-	static pfnGetFileInformationByHandleEx pGetFileInformationByHandleEx = NULL;
-	if (!pGetFileInformationByHandleEx) {
-		pGetFileInformationByHandleEx = (pfnGetFileInformationByHandleEx)
-			GetProcAddress(GetModuleHandle(TEXT("kernel32.dll")), "GetFileInformationByHandleEx");
-		if (!pGetFileInformationByHandleEx)
-			return -1;
-	}
-
-	nameinfo = malloc(size);
-	if (nameinfo == NULL) {
-		return -1;
-	}
+	nameinfo = alloca(size);
```

Comment:
Why not, alloca is a bit shorter, but the pairs of malloc and free seems correct here...
I'd rather consider this change as un-necessary.